doc/rados/configuration/pool-pg-config-ref.rst

```diff
@@ -181,8 +181,8 @@ Ceph configuration file.
 ``osd_pool_default_min_size``
 
 :Description: Sets the minimum number of written replicas for objects in the
-             pool in order to acknowledge a write operation to the client.  If
-             minimum is not met, Ceph will not acknowledge the write to the
+             pool in order to acknowledge an I/O operation to the client.  If
```

Comment:
i don't think min_size applies to read requests. we only reply the client with a `MOSDBackoff` message or simply wait for the recovery of the object in question when serving a write or a write-ordered client requests. but when it comes to a read request, there is no need to have min_size of replicas or shards to serve it.

Reply:
it applies to reads too, we don't let a pg go active (and thus it does not serve reads) below min_size. backoffs are added in PrimaryLogPG::do_request() if the pg is not active.

If we allowed reads before we went active, we might be serving clients out of date or divergent data.